libs/storagemapvec/src/lib.sw

```diff
@@ -81,7 +81,14 @@ impl<K, V> StorageMapVec<K, V> {
         store(len_key, len - 1);
 
         let key = sha256((key, len - 1, __get_storage_key()));
-        Option::Some::<V>(get::<V>(key))
+        match get::<V>(key) {
+            Option::Some(value) => {
+                Option::Some(value)
+            },
+            Option::None => {
+                Option::None
+            }
+        }
```

Comment:
Can this be cleaner? Similarly line 130

Reply:
tbh not sure what I was thinking here. Updated to just `get::<V>(key)`